libs/libxx/Kconfig

```diff
@@ -35,10 +35,11 @@ config LIBCXXMINI
 
 config LIBCXX
 	bool "LLVM libc++ C++ Standard Library"
+	depends on CXX_STANDARD="c++17" || CXX_STANDARD="c++20"
```

Comment:
does Kconfig support string comparison? And it will stop work after https://github.com/apache/nuttx/pull/8187. So I think it's enough to point the standard requirement in help text.

Reply:
It does work, for me at least. However, if #8187 gets integrated, this would break.